File: Java/Baju.java
class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju() {
        this.untuk = "";
        this.size = "";
        this.merk = "";
    }
    
    public Baju(String untuk, String size, String merk) {
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }
    
    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }
    
    public void setSize(String size) {
        this.size = size;
    }
    
    public void setMerk(String merk) {
        this.merk = merk;
    }
    
    public String getUntuk() {
        return this.untuk;
    }
    
    public String getSize() {
        return this.size;
    }
    
    public String getMerk() {
        return this.merk;
    }
}

File: CPP/Baju.cpp
#include "Aksesoris.cpp"

class Baju : public Aksesoris
{
    private:
        string untuk;
        string size;
        string merk;

    public:
        Baju(){
            this->untuk = "";
            this->size = "";
            this->merk = "";
        };
        
        Baju(string untuk, string size, string merk)
        {
            this->untuk = untuk;
            this->size = size;
            this->merk = merk;
        }
        
        void setUntuk(string untuk)
        {
            this->untuk = untuk;
        }

        void setSize(string size)
        {
            this->size = size;
        }

        void setMerk(string merk)
        {
            this->merk = merk;
        }

        string getUntuk()
        {
            return this->untuk;
        }

        string getSize()
        {
            return this->size;
        }

        string getMerk()
        {
            return this->merk;
        }
        
        ~Baju(){};
};
